### PR TITLE
NPM update

### PR DESCRIPTION
**Pull request by christopherdancy**
This PR completes the following:
1. Update NPM package
2. Redeploys contracts
3. Update dist/index file

### CODE REVIEW

**Review by herbig — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@fractal-framework/fractal-contracts",
-  "version": "0.1.8",
+  "version": "0.2.1",
```
Comment (tbwebb22) [inline package.json:3]: Why is this changed from 0.1.x to 0.2.x?

**Review by tbwebb22 — CHANGES_REQUESTED** [review]
